src/tests/multihost/sssd/testlib/common/remote_ssh.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"C"
+	"time"
+	"github.com/appleboy/easyssh-proxy"
```

Comment:
looks like it depends on above repo, who is maintaining them?, 
as in future if we get issue from above who is going to fix them?